### ISSUE TITLE
[Feature Request] core version list for FBA & MAME

### ISSUE THREAD

**Issue by crazii** [libretro/RetroArch#8793]
Scenario:
When I updated the FBA core, the new version diffs from my romset, some of my roms are not working anymore and it may be hard to find the latest version roms.
So I want to rollback fba core to an older version. But there's no way to do that.

Cores like MAME&FBA may need to update roms when they update to new version. But it should be optional, not forcing users to do that.  So a sub list for the history versions will be more user-friendly.

That may involve front end changes, build history reserve. it'll be more complicated if each version needs special bug fix, which need multiple code bases...

**Comment by i30817** [comment]
Far better to warn games will not scan/work on a new version. This is just as troublesome because you'd have to track upstream changes, so it's also not likely to happen.

**Comment by fr500** [comment]
@crazii we don't keep historic cores, nor stable cores. It's too much of a hassle.
Casually though I coded this last night.

https://git.retromods.org/radius/RetroArch/commit/02cf324a22c06d6bb28eb885027731fc59a4228e

**Comment by crazii** [comment]
@fr500 Cool, backup is working for me too. Guess I better keep my hands away from the update of cores.

**Comment by crazii** [comment]
@i30817 Yep warnings would also keep me out of this awkward trouble. I didn't know the "core update" will update the version, only thought it would push bug fixes.  It's too late when I knew. :)
